permissions/middleware.go

```diff
@@ -64,13 +70,57 @@ func AuthenticationMiddleware() gin.HandlerFunc {
 				}
 
 				c.Set("ID_TOKEN_CLAIMS", claims)
+
```

Comment:
as I understand we want add all users that call mdb, right? Ore make other middleware for labels calls?